.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+      "dbaeumer.vscode-eslint"
```

Comment:
Recommends [ESLint ](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) plugin in vscode